### ISSUE TITLE
Question about Partial re-training/fine-tuning

### ISSUE THREAD

**Issue by AmolKodange12** [jimazeyu/GraspSplats#24]
Hi,

Thanks so much for all your work and the amazing paper and resources!

I wanted to ask, the paper mentions :

> Partial Fine-Tuning. Edited scenes may have undesirable artifacts for regions unobserved during
> the initial reconstruction (e.g., surface underneath displaced objects). Optionally, GraspSplats sup-
> ports partial scene re-training using object masks rendered before and after the displacement, which
> is much more efficient than a complete reconstruction.
> 

But there's no mention of this in the readme. Is there some code that implements this? I would really like to know as it's a straightforward solution to occlusions during initial reconstruction.

Thanks and kind regards,
Amol Kodange
